pom.xml

```diff
@@ -56,6 +56,13 @@
       <version>2.0.4</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>matrix-auth</artifactId>
+      <version>1.3</version>
+      <scope>test</scope>
```

Comment:
Not exactly happy to see this come back, but I guess in `test` scope it is OK, and you needed two distinct `AbstractFolderProperty` implementations I suppose?